lib/PHPExif/Adapter/Exiftool.php

```diff
@@ -201,6 +218,26 @@ public function mapData(array $source)
             Exif::TITLE                 => (!isset($source['Title'])) ? false : $source['Title'],
             Exif::VERTICAL_RESOLUTION   => (!isset($source['YResolution'])) ? false : $source['YResolution'],
             Exif::WIDTH                 => (!isset($source['ImageWidth'])) ? false : $source['ImageWidth'],
+            Exif::GPS                   => $gpsLocation,
         );
     }
+
+    /**
+     * Extract GPS coordinates from formatted string
+     *
+     * @param string $coordinates
+     * @return array
+     */
+    protected function extractGPSCoordinates($coordinates)
```

Comment:
Can you unit test this method? Coverage is brought down now, which I want to avoid.